tests/bitmap.rs

```diff
@@ -0,0 +1,66 @@
+use ttf_parser::{RasterGlyphImage, RasterImageFormat};
+
+static FONT_DATA: &[u8] = include_bytes!("fonts/bitmap.otb");
```

Comment:
Add a comment that it's you own font and it has free license. CC0 I guess.

Reply:
Done